src/foam/comics/DAOUpdateControllerView.js

```diff
@@ -31,14 +33,30 @@ foam.CLASS({
   ],
 
   exports: [
+    'controllerMode',
     'data'
   ],
 
   css: `
+    ^ {
+      width: 1024px;
+      margin: auto;
+    }
+    ^action-container {
+      display: flex;
+      justify-content: space-between;
+      margin: 8px 0;
+    }
+    ^detail-container {
+      overflow-x: scroll;
+    }
+    /* TODO: Remove references to nanopay */
     ^ .net-nanopay-ui-ActionView {
       background: #59aadd;
       color: white;
-      margin-right: 4px;
+    }
+    ^ .net-nanopay-ui-ActionView + .net-nanopay-ui-ActionView {
```

Comment:
We shouldn't have nanopay CSS in foam code.

Reply:
Fixed in 69c1837.